src/app/components_ngrx/work-item-quick-preview/work-item-quick-preview.component.ts

```diff
@@ -227,7 +229,7 @@ export class WorkItemQuickPreviewComponent implements OnInit, OnDestroy {
       .subscribe(renderedHtml => {
         callBack(
           rawText,
-          renderedHtml
+          this.sanitizer.bypassSecurityTrustHtml(renderedHtml)
         );
       })
   }
```

Comment:
We are not using this component anymore. So, can you please remove this whole file.

Reply:
Done.